templates/terraform/resource.erb

```diff
@@ -129,13 +129,13 @@ func resource<%= resource_name -%>Create(d *schema.ResourceData, meta interface{
 	defer mutexKV.Unlock(lockName)
 <% end -%>
 
-	url, err := replaceVars(d, config, "<%= build_url(object.collection_url) -%>")
+	url, err := replaceVars(d, config, "<%= build_url(object.full_create_url) -%>")
 	if err != nil {
 		return err
 	}
 
 	log.Printf("[DEBUG] Creating new <%= object.name -%>: %#v", obj)
-	res, err := Post(config, url, obj)
+	res, err := <%= object.create_verb.to_s.capitalize -%>(config, url, obj)
```

Comment:
At this point, since we have customizable verbs for pretty much everything, I think we should go ahead and get rid of these separate methods and just use `sendRequest`

Reply:
I might do this in a seperate PR, if that's ok. I think it'll be nicer if the changes in Terraform are linked to a different PR, since having each resource switch from Post/Get/etc() to sendRequest() is a fairly large (file-wise) change.

Reply:
Oh very good point, thank you for that :)